frontend/index.html

```diff
@@ -37,7 +38,7 @@
   <!-- We un-hide the form using inline JS so that (when JS is enabled)
        it shows in the normal rendering order as if it was never hidden. -->
   <script>document.getElementById("domain-form").classList.remove("hidden");</script>
-  <noscript>Submitting entries to the HSTS preload list via this site requires Javascript.</noscript>
+  <noscript>Submitting entries to the HSTS preload list via this site requires JavaScript.</noscript>
```

Comment:
I personally refuse to capitalize it as `JavaScript`, but I guess the time of reckoning has come for this project. :-(

Reply:
I didn't do it either in the past, but https://en.wikipedia.org/wiki/JavaScript. So I guess it's the "official" name. :)